### DISCUSSION TITLE
Repository Deployment

### DISCUSSION THREAD

**Discussion by mabg1192** [community/community#55340] [New to GitHub]
Hello everyone, 
Getting started with github and deploying my first repository. After a successful build and deployment, unfortunately my page is not loading. Used Angular CL and everything works fine in my local machine. Here is the repo https://github.com/mabg1192/EFB
My guess is that index.html is the problem. Thank you in advance.

Edit:
Deploying the repository under https://mabg1192.github.io/ works. Cant figure out why https://mabg1192.github.io/EFB/ does not load since they are the same. Any idea? Thank you

**Comment by radar-blip** [comment]
Both are working. You must have figured something out. 👍🏻

**Comment by ademakdogan** [comment]
It's possible that the issue is related to the base href in your index.html file. When deploying to a subdirectory, you need to update the base href accordingly.

In your index.html file, try updating the base href tag to:


`
base href="/EFB/"
`

This will tell the browser to load all the assets relative to the /EFB/ directory.

Also, make sure that the generated files from the build are correctly pushed to the gh-pages branch on your GitHub repository. You can check this by going to the Settings tab on your repository and scrolling down to the GitHub Pages section. There should be a message indicating that your site is published at the https://mabg1192.github.io/EFB/ URL. If it's not there, make sure that the gh-pages branch is up to date with your latest changes.

**Reply by mabg1192** [reply]
Thank you for the reply. I will take a look at the index.html regarding the href.
Also, gh-pages branch has to be created to push the build under it? At the moment the page is deployed using the main branch.

Thank you once again.
